scipy/stats/_stats_py.py

```diff
@@ -7501,48 +7501,86 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
            in the case of a correlated system of variables is such that it can be reasonably
            supposed to have arisen from random sampling", Philosophical Magazine. Series 5. 50
            (1900), pp. 157-175.
+    .. [4] Mannan, R. William and E. Charles. Meslow. “Bird populations and
+           vegetation characteristics in managed and old-growth forests,
+           northeastern Oregon.” Journal of Wildlife Management
+           48, 1219-1238, :doi:`10.2307/3801783`, 1984.
 
     Examples
     --------
-    When just `f_obs` is given, it is assumed that the expected frequencies
-    are uniform and given by the mean of the observed frequencies.
+    In [4]_, bird foraging behavior was investigated in an old-growth forest
+    of Oregon.
+    In the forest, 44% of the canopy volume was Douglas fir,
+    24% was ponderosa pine, 29% was grand fir, and 3% was western larch.
+    The authors observed the behavior of several species of birds, one of
+    which was the red-breasted nuthatch. They made 189 observations of
+    foraging of this species noting that 23% of birds favoured Douglas
+    fir, 27% ponderosa pine, 29% grand fir, and 21% western larch.
+
+    Using a chi-square test, we can test the null hypothesis that the
+    proportions of foraging events are equal to the proportions of canopy
+    volume. As the authors of the paper, let's consider a significance level
+    of 1%.
```

Comment:
"As the author's of the paper, let's consider" con be interpreted as meaning that _we_ are the authors of the paper, so we get to choose the significant level. 

```suggestion
    volume. The authors of the paper considered a p-value less than 1% to be
    significant.
```